src/pages/Main.jsx

```diff
@@ -1,7 +1,79 @@
+import { Stack } from "@mui/system";
+import { styled } from "@mui/material";
+import Typography from "@mui/material/Typography";
```

Comment:
```suggestion
import Typography from "@mui/material/Typography";

```
외부 라이브러리, 내부 라이브러리 import 개행 필요